.scalafix.conf

```diff
@@ -1,9 +1,27 @@
 rules = [
-    RemoveUnused
+    RemoveUnused,
+    DisableSyntax,
+    ProcedureSyntax,
+    NoValInForComprehension,
+    LeakingImplicitClassVal
 ]
 
 RemoveUnused.imports = true
 RemoveUnused.privates = false
 RemoveUnused.locals = true
 RemoveUnused.patternvars = false
 RemoveUnused.implicits = false
+
+DisableSyntax.noVars = false
+DisableSyntax.noThrows = false
+DisableSyntax.noNulls = false
+DisableSyntax.noReturns = false
+DisableSyntax.noWhileLoops = false
+DisableSyntax.noAsInstanceOf = false
+DisableSyntax.noIsInstanceOf = false
+DisableSyntax.noXml = false
+DisableSyntax.noDefaultArgs = false
+DisableSyntax.noFinalVal = true
+DisableSyntax.noFinalize = true
+DisableSyntax.noValPatterns = false
+DisableSyntax.noUniversalEquality = false
```

Comment:
You seem to disable a lot of syntaxes here... Why? Is there some syntaxes we might wanna check?

Reply:
I tried to enable some config, but the result is not my expectations. It makes James code wrong. 
So I think the first shoot is a simple config.

Reply:
A bit surprised by your reasoning here. It's exactly because we spent a long time doing scala without a checkstyle that there might likely been a lot of wrong things in James.

If you do tell me though you want to solve them step by step in other PRs then I'm fine